e2e/libs/backing_image/crd.py

```diff
@@ -0,0 +1,91 @@
+from kubernetes import client
+from datetime import datetime
+from backing_image.base import Base
+
+from utility.utility import logging
+from utility.utility import get_retry_count_and_interval
+import time
+
+class CRD(Base):
+    def __init__(self):
+        self.obj_api = client.CustomObjectsApi()
+        self.retry_count, self.retry_interval = get_retry_count_and_interval()
+        
+    def create(self, bi_name, source_type, url, expected_checksum):
+        return NotImplemented
+
+    def get(self, bi_name):
+        return NotImplemented
+
+    def all_disk_file_status_are_ready(self, bi_name):
+        return NotImplemented
+    def clean_up_backing_image_from_a_random_disk(self, bi_name):
+        return NotImplemented
+
+    def delete(self, bi_name):
+        return NotImplemented
+
+    def wait_for_backing_image_disk_cleanup(self, bi_name, disk_id):
+        return NotImplemented
+
+    def wait_for_backing_image_delete(self, bi_name):
+        return NotImplemented
+
+    def cleanup_backing_images(self):
+        return NotImplemented
+    
+    def list_backing_image_manager(self):
+        label_selector = 'longhorn.io/component=backing-image-manager'
+        return self.obj_api.list_namespaced_custom_object(
+            group="longhorn.io",
+            version="v1beta2",
+            namespace="longhorn-system",
+            plural="backingimagemanagers",
+            label_selector=label_selector)
```

Comment:
@yangchiu Are we maintaining strings separately anywhere something like string.resource? It will make modifying and maintaining these commonly used strings easier later at one place, WDYT?